db/migrations/versions/009_add_binary_scoring_system.py

```diff
@@ -0,0 +1,31 @@
+"""Adds a binary scoring system to allow a simple accept/reject option
+
+Revision ID: "009_add_binary_scoring_system"
+Revises: "008_extend_flag_for_change_req"
+Create Date: 2024-12-19 16:30:00.346479
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "009_add_binary_scoring_system"
+down_revision = "008_extend_flag_for_change_req"
+branch_labels = None
+depends_on = None
+
+zero_to_one_id = "40d10744-b0c1-4af0-9cb9-df1058a3e930"
```

Comment:
I also think that we shouldn't be hardcoding IDs here, as that can lead to then just hardcoding the IDs in the codebase.

I think it's safer and leads to more robust code if the IDs are random in all environments. That means we then need to be able to pull the record out of the DB by some other identified, such as a descriptive name, and leads to better relational data and access patterns.

Reply:
Yeah that's true! I've leave it to be autogenerated.